src/pages/Admin/Admin.styles.ts

```diff
@@ -12,7 +12,6 @@ export const Wrapper = styled.div<WrapperProps>`
     hasSelectedVenue ? "240px auto" : "auto"};
 
   position: relative;
-  overflow-y: scroll;
```

Comment:
Why did we remove this here? Is it inherited from somewhere else? And/or guaranteed never to need to scroll/etc?

Reply:
It's removed because WithNavigation has overflow-y and this one was rendering another one which was creating a scroll inside of a scroll